ui/src/components/project/ReferenceAttributeForm.vue

```diff
@@ -3,44 +3,45 @@
     <header class="modal-card-head">
       <h3 class="modal-card-title">
         {{ title }}
-        <TableTag :table="table" class="is-large" />
       </h3>
     </header>
-    <section class="modal-card-body">
-      <b-field label="Table">
-        <b-select v-model="attribute.referencedTableId">
-          <option v-for="table in tables" :key="table.id" :value="table.id">{{ table.name }}</option>
+    <section class="modal-card-body reference-form">
+      <p>
+        On table <TableTag :table="table" class="is-medium" />,
+      </p>
+      <p>
+        create a link to the <label for="referenced-table"><strong>category table</strong></label>
```

Comment:
Hm, so we're renaming the entire `Dimension table` concept with the term `Category table`?

Reply:
Yeah we decided on "Dimension" -> "Category" and "Fact" -> "Measurements". Also "Project" -> "Cube". Will do the renaming in a separate PR.

Reply:
I mentioned this in chat, this is still aligned with Wikipedia entries of cubes but it's closer to the target audience. API will stay the way it is but this is for the users itself.

Reply:
> API will stay the way it is but this is for the users itself.

this separation can easily lead to errors and confusion. at some point we should unify the terminology across the board (UI, API, code, internal vocabulary)